common/app/views/fragments/containers/facia_cards/commercialContainer.scala.html

```diff
@@ -104,7 +104,16 @@ <h3 class="paidfor-popup__title">Paid stories are paid for and controlled by an
                                 </div>
                             </div>
                             <h3 class="commercial__title">@container.displayName</h3>
-                            <a class="commercial__cta" href="@LinkTo("/guardian-labs")">@inlineSvg("glabs-logo", "logo")</a>
+                            <a class="commercial__cta"
+                                @if(Edition(request).id == "AU") {
+                                    href="@LinkTo("/guardian-labs-australia")"
+                                } else {
+                                    href="@LinkTo("/guardian-labs")"
+                                    }
+                            >
+                                @inlineSvg("glabs-logo", "logo")
+                                <span class='u-h'>Guardian Labs</span>
```

Comment:
We could embed a title in the SVG itself, but I can think of two drawbacks:
1. We probably always want a title on our links, so that ATs read them out promptly when a user tabs over navigation points on the page. Adding an SVG title as well would mean inundating an AT user with repetitive information.
2. Embedding a title in our SVGs might make dynamic internationalization harder.